### DISCUSSION TITLE
Secret Region

### DISCUSSION THREAD

**Discussion by hboursiq** [prowler-cloud/prowler#5515] [General]
What would be the modifications required to get prowler to work from a docker container in AWS Secret Regions (no internet access)?

**Comment by MrCloudSec** [comment]
Hi Herold,

In this case you would have to manually update the [aws_regions_by_service.json](https://github.com/prowler-cloud/prowler/blob/master/prowler/providers/aws/aws_regions_by_service.json) and include the partition, region and services, you can find our documentation [here](https://docs.prowler.com/projects/prowler-open-source/en/latest/tutorials/aws/regions-and-partitions/#aws-iso-us-europe).

Let me know if that works for you!

Thanks.
